### PR TITLE
feat: update layout for ledger tx

### PR DESCRIPTION
**Pull request by euharrison**
Update layouts for:

- /approvals.html#/confirm-ledger-tx
- /approvals.html#/confirm-sign-tx

Screenshot 2024-07-26 at 11 33 34

Screenshot 2024-07-26 at 11 35 02

### CODE REVIEW

File: apps/extension/src/Approvals/ConfirmSignTx.tsx [hunk 1]
```diff
@@ -46,7 +48,6 @@ export const ConfirmSignTx: React.FC<Props> = ({ details }) => {
 
         setStatus(Status.Completed);
       } catch (e) {
-        console.info(e);
```
Comment (euharrison) [inline apps/extension/src/Approvals/ConfirmSignTx.tsx:49]: lint was complaining about this console, let me know if it need to be here that I add an lint ignore comment

Comment (jurevans) [reply apps/extension/src/Approvals/ConfirmSignTx.tsx:49]: we can remove it!

**Comment by jurevans** [comment]
@euharrison This works well for Ledger! I did notice a little discrepancy between button styles between the detail view & the view on submit, where the rounded corners are sharper on Details:

Screen Shot 2024-07-26 at 11 39 23 AM

Screen Shot 2024-07-26 at 11 39 30 AM

I think this is a problem with Tx Details though, since everywhere else they have more radius. There is a discrepancy in the Figma designs too, so not an issue with your PR!

**Review by jurevans — APPROVED** [review]
LGTM! Works well with keys & Ledger! I left one comment about the button corners when compared to the Details view, but I see that we should probably make this consistent throughout the extension anyways, so that can be handled separately.

**Comment by euharrison** [comment]
@jurevans thanks for noticing it!

Border radius fixed :) 

Screenshot 2024-07-29 at 10 22 52

Screenshot 2024-07-29 at 10 23 03

As your PR wasn't merged to main yet, do you think it's better if you press the merge button here?
Then you have a better control to decide the best moment to avoid conflicts for yourself

**Comment by jurevans** [comment]
> As your PR wasn't merged to main yet, do you think it's better if you press the merge button here? Then you have a better control to decide the best moment to avoid conflicts for yourself

nice! Yeah, I ended up having to make some changes, so I can handle that once everything is confirmed to work! Thanks!

**Comment by jurevans** [comment]
Merged these changes to #815  via another branch, thanks!

**Pull request closed without merging**